prog/kubernetes/provision_kubernetes_node.rb

```diff
@@ -134,7 +148,32 @@ def write_hosts_file_if_needed(ip = nil)
       nap 10
     when "Failed"
       # TODO: Create a page
-      Clog.emit("JOIN CLUSTER FAILED")
+      Clog.emit("JOIN CP NODE TO CLUSTER FAILED")
```

Comment:
all caps is kinda weird. also, would you want there to be anything as structured data, which is a hash returned from a block?